ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java

```diff
@@ -67,6 +66,8 @@ public void contactInfoTest() throws Exception {
         patientDashboardPage = activeVisitsPage.goToPatientDashboardOfLastActiveVisit();
         patientDashboardPage.clickOnShowContact();
         assertTrue(patientDashboardPage.getTelephoneNumber().contains("111111111"));
+        homePage = patientDashboardPage.goToHomePage();
```

Comment:
Once we have the presence of the telephone number  in the db asserted, does the test need to go back to home page? I think once the test asserts the number then it should then go to teardown() function.

Reply:
True we can let it finish from assert. Thanks.

Reply:
Resolved thanks.